src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -22,10 +26,13 @@ export interface AbstractChart {
 
 export interface IChartContainerProps {
     chartMeta: ChartMeta;
-    dataCache: StudyViewClinicalDataCountsCache;
-    filter: StudyViewFilter;
-    onUserSelection: (chartMeta: ChartMeta, value: string[]) => void;
-    onDeleteChart: (uniqueKey: string) => void;
+    promise: MobxPromise<any>;
+    filters: any;
+    onUserSelection?: any;
+    onResetSelection?: any;
```

Comment:
`onResetSelection` could be same as `onUserSelection`

Reply:
See above.